subcommands/targets/image.go

```diff
@@ -0,0 +1,31 @@
+package targets
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
```

Comment:
This all looks good. The only nit - we usually organize imports with core libraries, then a blank line, and then external deps. eg:
~~~
	"fmt"
        "os"

	"github.com/sirupsen/logrus"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"
~~~

Reply:
updated via a forced push.